CMakeLists.txt

```diff
@@ -58,6 +58,9 @@ add_custom_target(git-info BYPRODUCTS ${CMAKE_BINARY_DIR}/geninclude/version.cpp
 # And include JUCE
 add_subdirectory(libs/JUCE)
 
+# Optional variables
+set(SC3_INCLUDE_ASIO_SUPPORT FALSE)
```

Comment:
For reference: Set this to `TRUE` to enable ASIO for Windows standalone EXE. This is left off by default due to concerns with the Steinberg License terms for usage of the SDK.